force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,93 @@
+<template>
+    <lightning-card title="PD1 Calculator">
+
+        <!-- Process -->
+
+        <lightning-button
```

Comment:
```suggestion
    <lightning-card title="PD1 Calculator">
        <lightning-button
```

I would recommend not so many spaces, it's ok to do 1 line to break up parts of the components HTML, but you can also use your comments to do that as well.